modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/user/mgt/UserImportLoggingTestCase.java

```diff
@@ -0,0 +1,244 @@
+import java.rmi.RemoteException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import javax.activation.DataHandler;
+import javax.activation.FileDataSource;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.AUDIT_LOG_FILE_NAME;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.BULK_USER_IMPORT_ERROR_FILE;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.BULK_USER_IMPORT_OP;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.BULK_USER_IMPORT_SUCCESS_FILE;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.CAUSE;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.COUNT;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.DUPLICATE_USERNAME;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.DUPLICATE_USERS;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.DUPLICATE_USER_COUNT;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.ENCODING;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.FAILED_CAUSE;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.FAILED_USERNAME;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.FAILED_USERS;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.FAILED_USER_COUNT;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.LOG_FILE_NAME;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.PIPE_REGEX;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.RESULT_REGEX;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.SUCCESS_COUNT;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.SUCCESS_USER_COUNT;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.SUCCESS_USER_COUNT_IN_MIXED_MODE;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.USERS;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.USER_NAME;
+import static org.wso2.identity.integration.test.utils.BulkUserImportConstants.USER_STORE_DOMAIN;
+
+/**
+ * Test cases for the bulkUserImport log appender.
+ * This will check the logs are directed to the new file.
+ */
+public class UserImportLoggingTestCase extends ISIntegrationTest {
+
+    private static final Log log = LogFactory.getLog(UserImportLoggingTestCase.class);
+    private UserManagementClient userMgtClient;
+    private boolean isFilePresent = false;
+    private final String BULK_USER_FILE = getISResourceLocation() + File.separator + "userMgt"
```

Comment:
let's order these such that constants will be defined first and then the class variables

Reply:
Done